### PR TITLE
Build multi-arch images

### PR DESCRIPTION
**Pull request by ajaypratap003**
Build multi-arch images

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -29,15 +29,19 @@ jobs:
...
-          docker tag $IMAGE_NAME:latest quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:latest &&
-          docker push quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:latest
\ No newline at end of file
+        run: |
+          IMAGE_TAG="$(git describe --exact-match --tags | sed 's/v//')"
+          ARTEMIS_TAG="artemis.$(podman image inspect $IMAGE_NAME:latest | grep -Po -m 1 '(?<=APACHE_ARTEMIS_VERSION=)[^"]*')"
+          podman login --username=${{ secrets.QUAY_USERNAME }} --password=${{ secrets.QUAY_PASSWORD }} quay.io
+          podman manifest push $IMAGE_NAME:latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:latest
+          podman manifest push $IMAGE_NAME:latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:$IMAGE_TAG
+          podman manifest push $IMAGE_NAME:latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:$ARTEMIS_TAG
```
Comment (brusdev) [inline .github/workflows/release.yml:47]: I think images with ARTEMIS_TAG are not required for the self-provisioning-plugin
```suggestion
          podman login --username=${{ secrets.QUAY_USERNAME }} --password=${{ secrets.QUAY_PASSWORD }} quay.io
          podman manifest push $IMAGE_NAME:latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:latest
          podman manifest push $IMAGE_NAME:latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:$IMAGE_TAG
```

**Review by brusdev — APPROVED** [review]
LGTM +1, can you squash the commits and add a new line at of `.github/workflows/release.yml` before merging it?

I see you are also adding a step to build snapshot images but your commit message doesn't reflect that.

**Comment by brusdev** [comment]
@ajaypratap003 I see `ERROR: failed to solve: process "/bin/sh -c npm install -g yarn && yarn install --network-timeout 500000 && yarn build" did not complete successfully: exit code: 1` at https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/actions/runs/5316008951/jobs/9625058267?pr=53

Could you convert it to draft, if you are still working on it?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -30,14 +30,25 @@ jobs:
...
+
+      - name: Build and Push the image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
+          no-cache: true
+          push: true
+          tags: ${{ secrets.QUAY_NAMESPACE }}/${{env.IMAGE_NAME}}:dev.latest, $(git describe --exact-match --tags | sed 's/v//')
```
Comment (brusdev) [inline .github/workflows/release.yml:54]: Why is this step causing the following error on my fork?
https://github.com/brusdev/activemq-artemis-self-provisioning-plugin/actions/runs/5330753167/jobs/9657764969
```
Error: buildx failed with: ERROR: invalid tag "$(git describe --exact-match --tags | sed 's/v//')": invalid reference format
```

File: src/brokers/broker-details/components/Overview/Overview.container.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { FC } from 'react';
-import * as _ from 'lodash';
+import * as _ from 'lodash-es';
```
[the remark this answers is not in the document]
Comment (ajaypratap003) [reply src/brokers/broker-details/components/Overview/Overview.container.tsx:2]: No.

File: .github/workflows/release.yml (hunk 2 above)
Comment (ajaypratap003) [reply .github/workflows/release.yml:54]: > $(git describe --exact-match --tags | sed 's/v//')

I didn't see this issue on PR.
It seems  `$(git describe --exact-match --tags | sed 's/v//')` tag format is not correct. I'll check it. Can you try to build without `$(git describe --exact-match --tags | sed 's/v//')` tag.

Comment (brusdev) [reply .github/workflows/release.yml:54]: Sure, I'll try another build when I'll review this PR again, in the mean time you could execute this workflow on your fork to reproduce and fix the issue.

Comment (ajaypratap003) [reply .github/workflows/release.yml:54]: Fixed the tag issue on fork repo.
release CI job is running on fork repo, changes will be add in PR when fork ci job will green.
https://github.com/ajaypratap003/activemq-artemis-self-provisioning-plugin/actions/runs/5336328861/jobs/9670860246

File: src/brokers/broker-details/components/Overview/Overview.container.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (brusdev) [reply src/brokers/broker-details/components/Overview/Overview.container.tsx:2]: I would avoid not related changes?

**Comment by brusdev** [comment]
I merged your pr on the main branch of my fork to test the workflows and I see the following issues:
 https://github.com/brusdev/activemq-artemis-self-provisioning-plugin/commits/main

- context is 1GB `#9 transferring context: 1.06GB 87.3s done` (https://github.com/brusdev/activemq-artemis-self-provisioning-plugin/actions/runs/5343109445/jobs/9685766432)
- ci and release workflows take more than 20 mins

https://github.com/brusdev/activemq-artemis-self-provisioning-plugin/actions/runs/5343109445/jobs/9685766432
https://github.com/brusdev/activemq-artemis-self-provisioning-plugin/actions/runs/5343115105/jobs/9685778763

File: src/brokers/broker-details/components/Overview/Overview.container.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (ajaypratap003) [reply src/brokers/broker-details/components/Overview/Overview.container.tsx:2]: yes

**Comment by ajaypratap003** [comment]
@brusdev As ARM64 image taking time for building, I had discussion with other folks about building ARM64 container image and this is known issue specially for UI build. 

AMD64 image building is fine and as we are not building ARM64 image on PR and It will build on `main` branch after merging PR. 

I checked it on my fork branch and build is fine but taking long time as known issue. 
https://github.com/ajaypratap003/activemq-artemis-self-provisioning-plugin/actions/runs/5358976860/jobs/9721974019

I think we can merge this PR.

**Pull request merged**